pkg/resources/grant_ownership_acceptance_test.go

```diff
@@ -323,6 +323,108 @@ func TestAcc_GrantOwnership_OnObject_Table_ToDatabaseRole(t *testing.T) {
 	})
 }
 
+func TestAcc_GrantOwnership_OnObject_Procedure_ToAccountRole(t *testing.T) {
+	databaseId := acc.TestClient().Ids.RandomAccountObjectIdentifier()
+	databaseName := databaseId.Name()
```

Comment:
nit: this name is used once, is there a good reason to extract it as a variable?

the same with schema

Reply:
Nope, it was copy pasted and I didn't notice - changed.